contracts/delegate-factory.md

```diff
@@ -7,39 +7,40 @@ A DelegateFactory deploys Delegate contracts. [View the code on GitHub](https://
 Create a new `DelegateFactory` contract.
 
 ```java
-constructor(ISwap _swapContract) public
+constructor(ISwap factorySwapContract, IIndexer factoryIndexerContract) public
 ```
 
-| Param           | Type    | Description                                          |
-| :-------------- | :------ | :--------------------------------------------------- |
-| `_swapContract` | `ISwap` | Instance of the swap contract used to settle trades. |
+| Param                    | Type    | Description                                                     |
+| :----------------------- | :------ | :-------------------------------------------------------------- |
+| `factorySwapContract`    | `ISwap` | Instance of the swap contract used to settle trades.            |
+| `factoryIndexerContract` | `ISwap` | Instance of the indexer contract the delegate will deploy with. |
 
 ## `createDelegate`
 
 Create a new Delegate contract. Implements `IDelegateFactory.createDelegate`.
 
 ```java
 function createDelegate(
-  address _delegateContractOwner,
-  address _delegateTradeWallet
+  address delegateContractOwner,
+  address delegateTradeWallet
 ) external returns (address delegateContractAddress)
 ```
 
 | Param                    | Type      | Description                                             |
 | :----------------------- | :-------- | :------------------------------------------------------ |
-| `_delegateContractOwner` | `address` | Address of the owner of the sender for rule management. |
-| `_delegateTradeWallet`   | `address` | Address of the wallet that holds funds to be traded.    |
+| `delegateContractOwner`  | `address` | Address of the owner of the sender for rule management. |
```

Comment:
`Address of the owner of the sender for rule management`
I feel like this is confusing and could be massively simplified, e.g.
`Address of the owner of the new delegate contract`